fluentd/run.sh

```diff
@@ -1,5 +1,14 @@
 #!/bin/bash
 
+OCP_OPERATIONS_PROJECTS=${OCP_OPERATIONS_PROJECTS:-"default openshift openshift-"}
+OCP_FLUENTD_TAGS=('journal.system.** system.var.log** audit.log**')
+for p in ${OCP_OPERATIONS_PROJECTS}; do
+    if [[ "${p}" == *- ]] ; then
+      p="${p}*"
+    fi
+    OCP_FLUENTD_TAGS+=("**_${p}_**")
+done
```

Comment:
Not sure this is causing any error, but these 2 are different.
```
echo "${OCP_FLUENTD_TAGS[@]}"
journal.system.** system.var.log** audit.log** **_default_** **_openshift_** **_openshift-*_**
```
```
echo "${OCP_FLUENTD_TAGS}"
journal.system.** system.var.log** audit.log**
```
And we may need "export OCP_FLUENTD_TAGS"?

Reply:
Changing the type of OCP_FLUENTD_TAGS from an array to a string separated by a space, OCP_FLUENTD_TAGS is likely passed to ruby ENV successfully.  But still .operations is not created...  I'm suspecting a ruby code may not be evaluated in <match ...>?
```
diff --git a/fluentd/run.sh b/fluentd/run.sh
index b904a68..cb8a716 100644
--- a/fluentd/run.sh
+++ b/fluentd/run.sh
@@ -1,12 +1,12 @@
 #!/bin/bash
 
 OCP_OPERATIONS_PROJECTS=${OCP_OPERATIONS_PROJECTS:-"default openshift openshift-"}
-OCP_FLUENTD_TAGS=('journal.system.** system.var.log** audit.log**')
+OCP_FLUENTD_TAGS="journal.system.** system.var.log** audit.log**"
 for p in ${OCP_OPERATIONS_PROJECTS}; do
     if [[ "${p}" == *- ]] ; then
       p="${p}*"
     fi
-    OCP_FLUENTD_TAGS+=("**_${p}_**")
+    OCP_FLUENTD_TAGS+=" **_${p}_**"
 done
 export OCP_FLUENTD_TAGS
```